primitives/genesis-builder/README.md

```diff
@@ -0,0 +1,3 @@
+Substrate genesis builder
+
+License: GPL-3.0-or-later WITH Classpath-exception-2.0
```

Comment:
Why GPL-3.0 and not Apache-2.0?

I also think that this license thing should be clarified. I see some components around licensed under GPL-3.0 and others under Apache-2.0.

Who decides? What is the rational?

Reply:
@bkchr ?

Reply:
done in: https://github.com/paritytech/substrate/pull/14131/commits/706f3f2ba2de9888d361b179edcd816f926d89d0

Reply:
makes sense. Ty